server/schedule/checker/replica_strategy.go

```diff
@@ -55,7 +56,8 @@ func (s *ReplicaStrategy) SelectStoreToAdd(coLocationStores []*core.StoreInfo, e
 	filters := []filter.Filter{
 		filter.NewExcludedFilter(s.checkerName, nil, s.region.GetStoreIDs()),
 		filter.NewStorageThresholdFilter(s.checkerName),
-		filter.NewSpecialUseFilter(s.checkerName),
```

Comment:
Could we keep this function as an alias of `filter.NewLabelConstaintFilter` rather than removing it totally? Otherwise, we need to write long code every time we need the same usage filter.

Reply:
use some variables to simplify the code, PTAL